docs/quick_tutorial/authentication.rst

```diff
@@ -35,7 +35,7 @@ Steps
 
        cd ..; cp -r view_classes authentication; cd authentication
 
-#. Add ``bcrypt`` as a dependency in ``authentication/setup.py``:
+#. Add ``bcrypt`` and ``pyramid[pyramid]`` as a dependency in ``authentication/setup.py``:
```

Comment:
what is `pyramid[pyramid]`? This is definitely not correct but I can't tell what it's even supposed to be.